frontend/src/pages/projects/screens/detail/storage/StorageSizeBars.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import {
+  Bullseye,
+  Progress,
+  ProgressMeasureLocation,
+  Split,
+  SplitItem,
+  Text,
+  Title,
+} from '@patternfly/react-core';
+import { PersistentVolumeClaimKind } from '../../../../../k8sTypes';
+import { getPvcTotalSize } from '../../../utils';
+
+type StorageSizeBarProps = {
+  pvc: PersistentVolumeClaimKind;
+};
+
+const StorageSizeBar: React.FC<StorageSizeBarProps> = ({ pvc }) => {
+  const inUseValue = '0Gi';
+  const maxValue = getPvcTotalSize(pvc);
+  const percentage = (parseFloat(inUseValue) / parseFloat(maxValue)) * 100;
+
+  return (
+    <>
+      <Title headingLevel="h4">Size</Title>
```

Comment:
FWIW, `Title` is not meant for this kinda functionality. Not everything is a title semantically.